src/nodes/cls_node.c

```diff
@@ -200,7 +199,7 @@ static __rte_always_inline rte_edge_t get_next_index(__rte_unused struct rte_nod
 			if (is_ipv6_nd(ipv6_hdr))
 				return CLS_NEXT_IPV6_ND;
 			df->l3_type = ntohs(ether_hdr->ether_type);
-			return CLS_NEXT_IPV6_LOOKUP;
+			return CLS_NEXT_CONNTRACK;
```

Comment:
Please don't tell me the design document for graph nodes needs redrawing again :D

Reply:
I will provide the updated design document in a follow-up PR.